content/source/docs/cloud/workspaces/agent.html.md

```diff
@@ -0,0 +1,175 @@
+
+## Before Install
+
+### Supported Operating Systems
+
+[Agents](https://releases.hashicorp.com/tfc-agent) currently only support 64 bit Linux operating systems. You can also run the agent within Docker using our official [Terraform Agent Docker container](https://hub.docker.com/r/hashicorp/tfc-agent).
+
+### Supported Terraform Versions
+
+Agents support Terraform versions 0.12 and above. Workspaces configured to use Terraform versions below 0.12 will not be able to select the agent-based execution mode.
+
+### Networking Requirements
+
+In order for an agent to function properly, it must be able to make outbound requests over HTTPS (port 443) to the Terraform Cloud application APIs. This may require perimeter networking as well as container host networking changes, depending on your environment. The IP ranges are documented in the [Terraform Cloud IP Ranges documentation](https://www.terraform.io/docs/cloud/architectural-details/ip-ranges.html).
+
+Additionally, the agent must also be able to communicate with any services required by the Terraform code it is executing. This includes the Terraform releases distribution service, [releases.hashicorp.com](https://releases.hashicorp.com) (supported by [Fastly](https://api.fastly.com/public-ip-list)), as well as any provider APIs. The services which run on these IP ranges are described in the table below.
+
+Hostname | Port/Protocol | Directionality | Purpose
+--|--|--|--
+app.terraform.io | tcp/443, HTTPS | Outbound | Polling for new workloads and providing status updates
+releases.hashicorp.com | tcp/443, HTTPS | Outbound | Updating agent components and downloading Terraform binaries
+archivist.terraform.io | tcp/443, HTTPS | Outbound | Blob Storage
+
+### Operational Considerations
+
+The agent is distributed as a standalone binary which can be run on any supported system. By default, the agent will run in the foreground as a long-running process that will continuously poll for workloads from Terraform Cloud. We strongly recommend pairing the agent with a process supervisor to ensure that it is automatically restarted in case of an error.
+
+Agents do not guarantee a clean working environment per Terraform execution. Each execution is performed in its own temporary directory with a clean environment, but references to absolute  file paths or other machine state may cause interference between Terraform executions. We strongly recommend that you write your Terraform code to be stateless and idempotent. You may also want to consider using [single-execution mode](#optional-configuration-single-execution-mode) to ensure your agent only runs a single workload.
+
+### Updating
+
+The agent will automatically receive updates for the agent only. Administrators are required to update the host operating system and all other software.
```

Comment:
recieve `application code updates` or something similar? On reflection this reads like it might refer to work items rather than the binary itself